protocol/src/main/java/org/opensearch/sql/protocol/response/format/JdbcResponseFormatter.java

```diff
@@ -38,6 +38,12 @@ protected Object buildJsonObject(QueryResult response) {
     response.getSchema().getColumns().forEach(col -> json.column(fetchColumn(col)));
     json.datarows(fetchDataRows(response));
 
+    // Fetch cursor
+    String cursor = response.getCursor();
+    if (cursor != null) {
+      json.cursor(cursor);
+    }
+
     // Populate other fields
     json.total(response.size())
         .size(response.size())
```

Comment:
TODO: Total should reference the number of results in the table, while size should be the number of results in this response